### PR TITLE
Create project_details.gs

### PR DESCRIPTION
**Pull request by siddhimahawar**

### CODE REVIEW

File: project_details.gs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/**
+* Get the details of the project
+* @param {String projectId of the GCP project
```
Comment (shikhagupta1212) [inline project_details.gs]: missing }

File: project_details.gs [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+* @return {Object} Project Details having name, Project Number, project Id and other details
```
Comment (shikhagupta1212) [inline project_details.gs]: your comment and functions names are not matching. Function stats that you are just retrieving project number. Also can you rename variable from projectId to cloudProjectId and so on

File: project_details.gs (hunk 1 above)
Comment (siddhimahawar) [reply project_details.gs]: Updated

File: project_details.gs (hunk 2 above)
Comment (siddhimahawar) [reply project_details.gs]: Updated the function name and variable name

File: project_details.gs [hunk 3]
```diff
@@ -1,10 +1,10 @@
 /**
 * Get the details of the project
-* @param {String projectId of the GCP project
+* @param {String} cloudprojectId of the GCP project
```
Comment (shikhagupta1212) [inline project_details.gs]: camelcase- cloudProjectId

**Review by shikhagupta1212 — APPROVED** [review]

**Pull request merged**